man/fi_peer.3.md

```diff
@@ -387,6 +387,9 @@ struct fi_ops_srx_owner {
                    size_t size, uint64_t tag, struct fi_peer_rx_entry **entry);
     int (*queue_msg)(struct fi_peer_rx_entry *entry);
     int (*queue_tag)(struct fi_peer_rx_entry *entry);
+	 void	(*foreach_unspec_addr)(struct fid_peer_srx *srx,
```

Comment:
indent is off

Reply:
Thansk! Fixed it in PR #8906 which has the API change commits. Will rebase here once that is merged.